tests/saw/spec/DRBG/DRBG.saw

```diff
@@ -304,6 +294,16 @@ let seed_spec n = do {
     crucible_return (tm {{ 0 : [32] }});
 };
 
+let mix_spec n = do {
+    (sp, keyp, s) <- drbg_state "drbg";
+    (psp, datap) <- alloc_blob_readonly n;
```

Comment:
I know you're just matching the other functions, but "sp", "s", "psp" etc are not particularly helpful variable names. Could we be more descriptive?

Reply:
I definitely don't disagree that the variable names are not descriptive. My grasp of the SAW syntax and proofs is minimal, and honestly I'm not sure what `s`, `sp`, and `psp` are supposed to stand for. I was able to make these updates mostly by virtue of the fact that `s2n_drbg_seed()` and `s2n_drbg_mix()` are almost identical functions, so I could copy the SAW spec.

Reply:
I updated the variable names for the added spec to (what I think are correct and) more descriptive names.